bin/run

```diff
@@ -79,6 +79,7 @@ init_connections() {
     gcp_conn=(
       "google_cloud_airflow_dataproc"
       "google_cloud_derived_datasets"
+      "google_cloud_shared_prod"
```

Comment:
Did you newly set up this connection?

I assume changing the conn_id changes what kube cluster this connects to. Do we have a `bq-load-gke-1` cluster in shared-prod? I had assumed we'd need to use the existing conn_id, but specify a different project_id parameter for the queries.

Reply:
> I had assumed we'd need to use the existing conn_id, but specify a different project_id parameter for the queries.

you are totally right. and it's `--billing-project` that we are missing.